datadog/resource_datadog_synthetics_test_test.go

```diff
@@ -347,12 +348,13 @@ resource "datadog_synthetics_test" "foo" {
 	]
 
 	locations = [ "aws:eu-central-1" ]
-	options = {
```

Comment:
Please keep at least one test still using `options`, and introduce new tests with `options_list`.